src/context/TxContext/index.tsx

```diff
@@ -50,11 +50,15 @@ function TxContextProvider({children}: PropTypes): React.ReactElement {
     const {api, txMethod, params, description, address, title} = config;
     const [section, method] = txMethod.split('.');
 
+    if (!section || !method) {
+      throw new Error('section or method undefined');
```

Comment:
How is this error being handled?

Reply:
it'll be caught by `start(...args).catch()` so no crashes. the check is basically identical to the next if check
```
if (!get(api.tx, [section, method])) {
```
TS needs this to understand `section` & `method` are not `undefined`.